tools/debug/debugsymbolviewer.py

```diff
@@ -19,9 +19,10 @@
 from __future__ import print_function
 from math import log
 import re
+import sys
 
-systemmap_path = '../../build/output/bin/System.map'		# System map file
-log_file = '../dump_tool/log_file'				# Crash log file
+systemmap_path = '../../build/output/bin/'		# System map file
```

Comment:
This path is not only for system map.
Let's rename it as output path or something general.

Reply:
Modified